source/Calamari.AzureCloudService/AuthHttpClientFactory.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Net;
+using System.Net.Http;
+
+namespace Calamari.AzureCloudService
+{
+    public class AuthHttpClientFactory
```

Comment:
Do we need this duplicated?

Reply:
It's duplicated in this LTS branch but has been consolidated into Azure.CloudAccounts in master and is shared from there. As this is just the case for this LTS branch I wasn't too phased adding the additional class, it's what is there currently.